ansible/group_vars/all

```diff
@@ -138,6 +140,9 @@ kafka:
   offsetsTopicReplicationFactor: "{{ kafka_offsetsTopicReplicationFactor | default(kafka_replicationFactor) | default((groups['kafkas']|length)|int) }}"
   # adapt this param for production deployments depending on the number of kafka consumers
   networkThreads: "{{ kafka_network_threads | default(3) }}"
+  # whisk.activation.payload.max + 5% of that value to account for data added by Kafka producer serialization
+  payload:
+    max: "{{ limit_activation_payload + (limit_activation_payload * 0.05) | int | default() }}"
```

Comment:
i think the 5% is not adequate in general. We can actually determine a fixed overhead because (all?) the fields have limits on size.